src/index.ts

```diff
@@ -1,9 +1,9 @@
 declare module '@webcomponents/custom-elements';
 
-(async function () {
-  if (!('customElements' in self)) {
-    await import('@webcomponents/custom-elements');
-  }
-
+if (!('customElements' in self)) {
```

Comment:
If this is fragile (as in, it needs to be written in this way else webpack or whatever doesn't do the right thing), add a comment stating that, else I'll probably break it again 😀